cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AuthenticationUtils.java

```diff
@@ -254,11 +255,11 @@ public static void configureClientAuthenticationEnvVars(KafkaClientAuthenticatio
                 properties.put(SASL_USERNAME, passwordAuth.getUsername());
                 properties.put(SASL_PASSWORD_FILE, String.format("%s/%s", passwordAuth.getPasswordSecret().getSecretName(), passwordAuth.getPasswordSecret().getPassword()));
                 properties.put(SASL_MECHANISM, KafkaClientAuthenticationPlain.TYPE_PLAIN);
-            } else if (authentication instanceof KafkaClientAuthenticationScramSha512) {
-                KafkaClientAuthenticationScramSha512 passwordAuth = (KafkaClientAuthenticationScramSha512) authentication;
+            } else if (authentication instanceof KafkaClientAuthenticationScram) {
+                KafkaClientAuthenticationScram passwordAuth = (KafkaClientAuthenticationScram) authentication;
```

Comment:
I know it was in the code before but `passwordAuth` sounds strange to me ... can we use something like `scramAuth` as we have the `tlsAuth` above? The same comments applies in all other places where we have this `passwordAuth` when it's SCRAM in reality.

Reply:
BTW: SCRAM is password based authentication.

Reply:
@scholzj I know that of course, but `passwordAuth` doesn't sound to be good enough to me.

Reply:
Was this renamed as @ppatierno requested?

Reply:
It is now.